docs/corobo.rst

```diff
@@ -0,0 +1,68 @@
+Using corobo
+============
+
+GitHub/GitLab utilities
+-----------------------
+
+1. Getting invitation to the org:
+   Send ``helloworld`` in the chat room.
+2. Want to work on an issue, first assign yourself:
+   ``corobo assign <issue-link>``
+3. Assigned issue is time-consuming, no longer interesting? Wanna discontinue:
+   ``corobo unassign <issue-link>``
+4. Wanna file a new issue, but don't want to leave gitter?:
+   ``corobo new issue <repo> <title>\n[description\n]+``
+5. PR is up, waiting for a review. Consider reviewing other PRs while you wait
+   for review of your PR. If the PR needs more work, mark it
+   work-in-progress(wip):
```

Comment:
This whole paragraph could be confusing for newcomers. Just mention about marking a PR as wip